src/main/java/core/basesyntax/service/impl/ParserServiceImpl.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ParserService;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ParserServiceImpl implements ParserService {
+    private static final int TYPE_INDEX = 0;
+    private static final int FRUIT_INDEX = 1;
+    private static final int QUANTITY_INDEX = 2;
+
+    @Override
+    public List<Fruit> parseData(List<String[]> fileInfo) {
```

Comment:
```suggestion
    public List<FruitTransaction> parseData(List<String[]> fileInfo) {
```
Нехай цей метод повертає список з транзакцій, які ми потім будемо по черзі обробляти за допомогою  OperationHandler. Наприклад для операції "s;banana;100" ми використаємо SupplyOperationHandler, який дістане з Storage фрукт з назвою banana і додасть до кількості, яка вже лежить в Storage ще 100. Так ми пройдемося по всім транзакціям і в кінці отримаємо в Storage дані про фрукти, які залишилися після всіх транзакцій